extensions/VisualEditor/wikia/modules/ve/ui/widgets/ve.ui.WikiaFocusWidget.js

```diff
@@ -33,7 +33,8 @@ ve.ui.WikiaFocusWidget = function VeUiWikiaFocusWidget( surface ) {
 	this.$body = this.$( this.getElementDocument() ).find( 'body:first' );
 	this.$window = this.$( this.getElementWindow() );
 	this.$surface = surface.$element;
-	this.$pageHeaderElements = this.$( '#WikiaPageHeader' ).children( ':not( h1 )' );
+	this.$pageHeaderElements = this.$( '#WikiaPageHeader' ).children( ':not( .header-container  )' ).add( '.header-tally' );
```

Comment:
why not add some class to WikiaPageHeader when VE is loaded and inside PageHeader.scss style things which should be hidden/visible.
This way all code related with styling PageHeader would be in one place.
Without knowing VE code root cause wasn't obvious.